coderedcms/utils.py

```diff
@@ -44,7 +44,7 @@ def convert_to_amp(value):
     Function that converts non-amp compliant html to valid amp html.
     value must be a string
     """
-    soup = BeautifulSoup(value)
+    soup = BeautifulSoup(value, "html5lib")
```

Comment:
Is there a reason this is using "html5lib" and coderedcms_tags.py is using different "html.parser"?

Reply:
They should both be using html.parser actually.  html5lib makes it valid html (ie includes body/head tags)